src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -2,6 +2,7 @@ package com.typesafe.sbt
 package packager
 package docker
 
+import java.io.File
```

Comment:
Still? I don't understand what it wants. Maybe it doesn't support Java packages? Or should it be above the package declarations?